### ISSUE TITLE
py-irt: command not found

### ISSUE THREAD

**Issue by naokishibuya** [nd-ball/py-irt#48]
Hi @EntilZha and @jplalor,

I want to report a potential problem with py-irt.

I tried with python3.10 and got the `py-irt: command not found`.

To reproduce:

```bash

python3.10 -m venv venv
source venv/bin/activate
pip install --upgrade pip
pip install torch pyro-ppl py-irt

py-irt

```

Then, I got `bash: py-irt: command not found`.

Related: https://github.com/nd-ball/py-irt/issues/17

**Comment by r-b-g-b** [comment]
I had the same issue: I think the problem is that version 0.2.1 (which implements the CLI) is not available for Python 3.10.

```bash
pip install py-irt==0.2.1
```

<details>
  <summary>Output</summary>

```
# ERROR: Ignored the following versions that require a different python version: 0.2.1 Requires-Python >=3.7,<3.10; 0.3.0 Requires-Python >=3.7,<3.10; 0.3.1 Requires-Python >=3.7,<3.10; 0.3.2 Requires-Python >=3.7,<3.10; 0.3.3 Requires-Python >=3.7,<3.10; 0.3.4 Requires-Python >=3.7,<3.10; 0.4.0 Requires-Python >=3.7,<3.10; 0.4.1 Requires-Python >=3.7,<3.10; 0.4.2 Requires-Python >=3.7,<3.10; 0.4.3 Requires-Python >=3.7,<3.10; 0.4.4 Requires-Python >=3.7,<3.10; 0.4.5 Requires-Python >=3.8,<3.10; 0.4.6 Requires-Python >=3.8,<3.10; 0.4.7 Requires-Python >=3.8,<3.10; 0.4.8 Requires-Python >=3.8,<3.10; 0.4.9 Requires-Python >=3.8,<3.10
ERROR: Could not find a version that satisfies the requirement py-irt==0.2.1 (from versions: 0.0.1, 0.0.2, 0.0.3, 0.1.1)
ERROR: No matching distribution found for py-irt==0.2.1
```
</details>

`Requires-Python >=3.7,<3.10`

**Comment by jplalor** [comment]
Thanks! version 0.4.10 can now be downloaded for Python 3.10 and 3.11
